lib/web/apiserver.go

```diff
@@ -1056,6 +1056,9 @@ func (h *Handler) githubCallback(w http.ResponseWriter, r *http.Request, p httpr
 	response, err := h.cfg.ProxyClient.ValidateGithubAuthCallback(r.URL.Query())
 	if err != nil {
 		logger.WithError(err).Error("Error while processing callback.")
+		if isRoleMatchError(err) {
```

Comment:
Any chance we could the change to this callback, in addition to isRoleMatchError?

Same for `oidcCallback` below.

Reply:
I think you lost a word! Can you clarify this?

Reply:
Sorry, I meant if we could test the change to the callback, in addition to the tests you added for isRoleMatchError.